e2e/support/constant.ts

```diff
@@ -0,0 +1 @@
+export const starterTemplatePluginId = 'com.mattermost.plugin-starter-template';
```

Comment:
Same question here. Can we extract the plugin ID?

Reply:
Same -- to do it manually with guides at the readme. This is base test only and will fail if not change.